composer.json

```diff
@@ -41,6 +42,7 @@
         "phpunit/phpunit": "8.5.*"
     },
     "suggest": {
+        "ext-mbstring": "For MultibyteTextNormalizer",
```

Comment:
We may use multibyte strings in other parts of the library so this might not always be the case and, if so, I guarantee I'll forget to change this line to something else. Perhaps we can make a more general statement such as "For fast multibyte string manipulation."

Open to other catchy phrases